src/templates/admin/submission/edit/author.html

```diff
@@ -49,6 +49,7 @@
         <h2>{% trans "Name, bio, and identifiers" %}</h2>
         {% if not form %}
           <form method="GET">
+            <input type="hidden" name="next" value="{{ request.GET.next }}">
```

Comment:
I see we guard against a value of `None` elsewhere in the code base. Should we do the same here to avoid `None` being serialized as text in the HTML? (i.e. `{{ request.GET.next|default:""}}`